src/samplers/per_operation_sampler.js

```diff
@@ -92,13 +92,19 @@ export default class PerOperationSampler implements Sampler {
 
   onCreateSpan(span: Span): SamplingDecision {
     const outTags = {};
-    const isSampled = this.isSampled(span.operationName, outTags);
-    // NB: return retryable=true here since we can change decision after setOperationName().
+    if (!span.context()._samplingState.isLocalRootSpan(span.context())) {
+      return { sample: false, retryable: false, tags: outTags };
+    }
+    let isSampled = this.isSampled(span.operationName, outTags);
+    // returning retryable=true since we can change decision after setOperationName().
```

Comment:
I think this is a bit confusing because we set retryable 'false' onSetOperationName - perhaps "the decision can change the first time setOperationName is called?"